openid-sharedsignals-framework-1_0.md

```diff
@@ -829,6 +829,10 @@ an HTTP POST request to the Configuration Endpoint. On receiving a valid request
 the Event Transmitter responds with a "201 Created" response containing a
 [JSON][RFC7159] representation of the stream’s configuration in the body.
 
+If the stream already exists, the Event Transmitter responds with HTTP status code
+"200 OK" and includes a [JSON][RFC7159] representation of the stream’s configuration
```

Comment:
```suggestion
"200 OK" and includes a [JSON][RFC7159] representation of the existing stream’s configuration
```

Reply:
Addressed in [9497085](https://github.com/openid/sharedsignals/pull/50/commits/94970859e67adfb2248a79703986d356fdc2cb04)